docs/native-modules-ios.md

```diff
@@ -1,442 +1,466 @@
 ---
 id: native-modules-ios
-title: Native Modules
+title: iOS Native Modules
 ---
 
-Sometimes an app needs to access a platform API and React Native doesn't have a corresponding module yet. Maybe you want to reuse some existing Objective-C, Swift or C++ code without having to reimplement it in JavaScript, or write some high performance, multi-threaded code such as for image processing, a database, or any number of advanced extensions.
+Welcome to Native Modules for iOS. Please start by reading the [Native Modules Intro](native-modules-intro) for an intro to what native modules are.
 
-We designed React Native such that it is possible for you to write real native code and have access to the full power of the platform. This is a more advanced feature and we don't expect it to be part of the usual development process, however it is essential that it exists. If React Native doesn't support a native feature that you need, you should be able to build it yourself.
+## Create a Calendar Native Module
 
-This is a more advanced guide that shows how to build a native module. It assumes the reader knows Objective-C or Swift and core libraries (Foundation, UIKit).
+In the following guide you will create a native module, `CalendarModule`, that will allow you to access Apple's calendar APIs from Javascript. By the end you will be able to call `CalendarModule.createCalendarEvent(‘Dinner Party’, ‘My House’);` from JavaScript, invoking a native method that creates a calendar event.
```

Comment:
Apostrophe characters are incorrect in the code here (which will lead to syntax errors). 

The typography correct version has been used (`‘` and `’`) instead of simple `'`.

Reply:
updated